docs/release/Betaflight-3.0-Release-Notes.md

```diff
@@ -2,7 +2,7 @@
 # BetaFlight 3.0 Release Notes
 
 This is really a major release. The full detailed change list can be found in the the commit history.
-[https://github.com/betaflight/betaflight/commits/master]
+[https://github.com/betaflight/betaflight/commits/master] 
```

Comment:
👾

Reply:
good catch, trailing space.  most editors can auto-fix this if configured. (configure for new/edited lines only, not untouched lines)

Reply:
at this point I'm going to throw VS Code out of the window(s)